tensorflow/python/keras/saving/model_config.py

```diff
@@ -31,9 +31,15 @@
 # pylint: enable=g-import-not-at-top
 
 
-@keras_export('keras.models.model_from_config')
+@keras_export('keras.models.from_config')
```

Comment:
Let's not change this, this would affect the public API.

Reply:
Done.